classes/class_animationplayer.rst

```diff
@@ -138,7 +138,7 @@ Member Variables
 Numeric Constants
 -----------------
 
-- **ANIMATION_PROCESS_PHYSICS** = **0** --- Process animation during the physics process. This is specially useful when animating physics bodies.
+- **ANIMATION_PROCESS_PHYSICS** = **0** --- Process animation during the physics process. This is especially useful when animating physics bodies.
```

Comment:
This one needs to modified in the source XML: https://github.com/godotengine/godot/blob/master/doc/classes/AnimationPlayer.xml#L384
We can still merge it, but it will be reverted next time we sync the docs if not fixed upstream.

Reply:
Upstream change merged! :smile:

godotengine/godot#12547